lesson4/backend28.js

```diff
@@ -5,14 +5,20 @@ app.listen(3618);
 app.use(express.static('public'));
 const bodyParser = require('body-parser');
 app.use(bodyParser.json({limit: '500mb'}));
+const gm = require('gm');
 
 app.post('/send',(req,res)=>{
   const firstData = req.body.name;
   const secondData = req.body.comment;
   const pic = req.body.newPicData;
-  fs.appendFile('/home/kinue/garageScript/lesson4/public/storing.txt',`${firstData} : ${secondData}\n`)
+  const storingTxt = '/home/kinue/garageScript/lesson4/public/storing.txt';
+  const memePath = '/home/kinue/garageScript/lesson4/public/MemePhotos/' + firstData + '.png';
+  
+  fs.appendFile(storingTxt,`${firstData} : ${secondData}\n`)
```

Comment:
What is this line for? Explain it here or personaly

Reply:
I'll explain personaly